src/themes.ts

```diff
@@ -96,6 +96,7 @@ export const light: PreciseFullTheme = {
   ui4: colors.lighterGray,
   ui5: colors.tuna,
   ui6: colors.transparent,
+  ui7: colors.grey4,
```

Comment:
Hm maybe let's do it like that. Use grey directly (grey is grey), but midnight via ui7. I am not sure if grey should be put on the theme as its also missing (like *all* greys) in the pattern library.